### PR TITLE
Javascript capstone 2

### PR DESCRIPTION
**Pull request by AjroudiRida**
- Use JavaScript to make websites dynamic and build basic single-page apps.
- Use ES6 syntax.
- Use ES6 modules.
- Use callbacks and promises.
- Use Webpack.
- Apply JavaScript best practices and language style guides in code.
- Follow Gitflow.
- Solve simple git conflicts.
- Send and receive data from an API.
- Use API documentation.
- I build The home page.
- When the page loads, I Involvement API to show the item likes.
- Add footer
- Create a comments popup
- Add new comment
- Add comment counter
- Display comments for a given item
- Adding testing files for likes count.
- Adding Comments counter unit testing

bellow there you can see a video explaining how the project work:
https://drive.google.com/file/d/1j0jxw68NcShAoEvubRVvz1NdmdHN4n5i/view?usp=sharing

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                </div>
+            </div>
+        </div>
+    </div>
+    <header class="header">
+        <img src="./assests/Images/logo.png" alt="SeaStar Catch & Grill" class="logo">
+        <nav class="nav">
+            <ul class="nav__list">
+                <li class="nav__item"><a href="#" class="nav__link">Home</a></li>
+                <li class="nav__item"><a href="#" class="nav__link">Meals</a></li>
```
Comment (Ismailco) [inline src/index.html]: - [ ] Counters should be present in all the interfaces that show the number of items (home), kindly make sure to add a counter for the home page that displays the number of meals
- [ ] The New counter should be covered with tests, kindly consider testing the meals counter

File: src/modules/function.js [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+      const ingred = `
+      <ion-icon class="ingredent-icon" name="restaurant-outline"></ion-icon><span class="ingredent-text">${ingredient}</span>
+    `;
+      const li = document.createElement('li');
+      li.setAttribute('class', 'ingredients--item');
+      li.innerHTML = ingred;
+      ingredientList.appendChild(li);
+    });
+
+    // const nbCmt = await commentsCounter(mealId);
```
Comment (Ismailco) [inline src/modules/function.js]: - [ ] Kindly consider removing commented code from your code base so it looks cleaner and easy to navigate

**Review by Ismailco — CHANGES_REQUESTED** [review]
#### Hi @AjroudiRida, @newtonah0, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️ Nace UI
✔️ Clean code
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by cynthiainga — APPROVED** [review]
Hi @AjroudiRida & @newtonah0, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] Good design ✔️ 
- [ ] All requested changes are implemented ✔️ 
- [ ] No linters errors ✔️ 
- [ ] All tests are passing ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**